Code/PgSQL/rdkit/mol_op.c

```diff
@@ -287,6 +285,23 @@ Datum mol_hash(PG_FUNCTION_ARGS) {
   PG_RETURN_CSTRING(res);
 }
 
+PGDLLEXPORT Datum mol_nm_hash(PG_FUNCTION_ARGS);
+PG_FUNCTION_INFO_V1(mol_nm_hash);
+Datum mol_nm_hash(PG_FUNCTION_ARGS) {
+  CROMol mol;
+  char *opts = PG_GETARG_CSTRING(1);
```

Comment:
Note: this call always had a default, so we don’t need to check if opts is an empty string.